### PR TITLE
#MM-34893 removed a section about Admin Advisor as the feature is now disabled

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
removed a section about Admin Advisor as the feature is now disabled

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-34893

### CODE REVIEW

File: source/overview/compliance.rst [hunk 1]
```diff
@@ -179,6 +179,9 @@ What information is shared when I click **Contact us** on a Mattermost Admin Adv
 
 Clicking **Contact us** in the Mattermost Admin Advisor will send some information to us. This may include the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number. This information is used to contact you as requested and to help us better understand your needs.
 
+.. note::
+    Mattermost `Admin Advisor notices are disabled <https://docs.mattermost.com/administration/notices.html#admin-advisor-notices>`__ in Mattermost Server from v5.35.
```
Comment (jasonblais) [inline source/overview/compliance.rst:183]:
```suggestion
    `Mattermost Admin Advisor notices are disabled <https://docs.mattermost.com/administration/notices.html#admin-advisor-notices>`__ in v5.35 and later.
```

**Review by jasonblais — CHANGES_REQUESTED** [review]

**Comment by harshilsharma63** [comment]
@jasonblais I've applied your changes.

**Review by jasonblais — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]
Thanks!

**Pull request merged**